tests/lib/mem_alloc/testcase.yaml

```diff
@@ -1,3 +1,7 @@
+common:
+    integration_platforms:
+      - qemu_cortex_m3
```

Comment:
Not familiar with this. Does this mean that the test will be only run on qemu_cortex_m3?

Reply:
Its means for CI purposes that we will only build/run on `qemu_cortex_m3`.

Reply:
Uhm, then I really prefer having this running on `qemu_cortex_a53` as well considering that we have recently fixed a few bugs AArch64-specific that we discovered when using newlibc.

Reply:
qemu_cortex_m3 is really very limited (no user mode, no MPU etc.) 
If you would like a cortex-m, please, choose mp2_an385.

Reply:
I dropped setting any platforms for this test, since its probably best it runs on each arch.